index.css

```diff
@@ -0,0 +1,171 @@
+body {
+    background-color: #524e4e;
+}
+h1 {
+    padding-left: 2%
+}
+header {
+    background-color: #df4646; 
+    padding: 2%;
+    font-size: 3em;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: left;
+    color:white;
+}
+/*this is the css for the NAVIGATION*/
+nav {
+    background-color: white;
+    font-family: Arial, Helvetica, sans-serif;
+    text-decoration: none;
+    display: flex;
+    justify-content: end;
+    background-color: #080808;
+}
+
+nav a {
+    padding-left: 15px;
+    padding-right: 15px;
+    color: white;
+    text-decoration-line: none;
+    font-size: 24px;
+}
+
+nav > div a:hover{
+    background-color: #df4646;
+    font-size: 26px;
+
+}
+
+/*this is the css for the main content*/

```

Comment:
Your well-named styles make this comment redundant -- consider removing it, keeping your code clean of unnecessary comments.